view/frontend/templates/js/boltjs.phtml

```diff
@@ -20,8 +20,7 @@
  *
  * @var $block \Bolt\Boltpay\Block\Js
  */
-if ($block->shouldDisableBoltCheckout()) return;
-
+if ($block->shouldDisableBoltCheckout("withproductpage")) return;

```

Comment:
This file is loaded on every page. The "withproductpage" parameter does not seem right.

Reply:
For now this file loaded on evert page IF minicart is enabled.
So if minicart is disabled we need to load this file only on product page, cart and checkout page.
But we cound instead parameter use new method and call it shouldDisableBoltScript(). Do you agree?

Reply:
@vitaliyreznikov my point was it already had worked for the two page types, cart and checkout, without the additional parameter. I guess it should not be much more complicated than adding product page to the list of default pages.